base/linalg/generic.jl

```diff
@@ -1268,27 +1318,34 @@ end
 end
 
 """
-    normalize(v, [p::Real=2])
+    normalize(v::AbstractVector, p::Real=2)
 
-Normalize the vector `v` with respect to the `p`-norm.
+Normalize the vector `v` so that its `p`-norm is equal to unity,
```

Comment:
Actually I'd rather keep "the vector", it's not that long and it's a useful information if you jump directly to the description without looking precisely at the signature.